lib/fastlane/actions/mailgun.rb

```diff
@@ -45,6 +45,11 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :to,
                                        env_name: "MAILGUN_TO",
                                        description: "Destination of your mail"),
+          FastlaneCore::ConfigItem.new(key: :from,
+                                       env_name: "MAILGUN_FROM",
+                                       optional: true,
+                                       description: "Mailgun Sandbox",
+                                       default_value: "fastlane build"),
```

Comment:
The default value for this should be the hardcoded value that is replaced, so "Mailgun Sandbox"